src/prisma/migrations/20240226232919_add_event_subscriptions/migration.sql

```diff
@@ -0,0 +1,54 @@
+/*
+  Warnings:
+
+  - Added the required column `maxBlocksToIndex` to the `configuration` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "configuration" ADD COLUMN     "indexerListenerCronSchedule" TEXT,
+ADD COLUMN     "maxBlocksToIndex" INTEGER NOT NULL;
```

Comment:
this will need a `DEFAULT` value if you mark it as `NOT NULL`.

Either mark it ass `nullable` or add a `default` value.

otherwise, get this error:

```
Database error:
ERROR: column "maxBlocksToIndex" of relation "configuration" contains null values

DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E23502), message: "column \"maxBlocksToIndex\" of relation \"configuration\" contains null values", detail: None, hint: None, position: None, where_: None, schema: Some("public"), table: Some("configuration"), column: Some("maxBlocksToIndex"), datatype: None, constraint: None, file: Some("tablecmds.c"), line: Some(6046), routine: Some("ATRewriteTable") }
```